gcdevops/OdooDocker
# generated translation values and module

resolves https://github.com/gcdevops/HRWhiteListing/issues/141

This PR seeks to add french translations for departments and job positions.

How to test 
- First completely wipe your local instance. I had to regenerate the csv's so that the id's do not have special characters 
- Import data as described in the readme 
- install the fr_CA language, make sure to have developer mode active and uncheck the override term setting as this causes an error for some reason 
- install the hr_translations. module
